homeassistant/strings.json

```diff
@@ -69,12 +69,12 @@
       },
       "abort": {
         "single_instance_allowed": "Already configured. Only a single configuration possible.",
-        "already_configured_account": "Account is already configured",
-        "already_configured_device": "Device is already configured",
-        "already_configured_location": "Location is already configured",
-        "already_configured_service": "Service is already configured",
-        "already_in_progress": "Configuration flow is already in progress",
-        "no_devices_found": "No devices found on the network",
+        "already_configured_account": "Account is already configured.",
+        "already_configured_device": "Device is already configured.",
+        "already_configured_location": "Location is already configured.",
+        "already_configured_service": "Service is already configured.",
+        "already_in_progress": "Configuration flow is already in progress.",
+        "no_devices_found": "No devices found on the network.",
```

Comment:
I don't think this is needed tbh.
This will cause thousands of PRs and invalidate all translations we have.

Let's not go down that path.

../Frenck